app/components/sidebars/settings/index.js

```diff
@@ -4,23 +4,31 @@
 import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
+import {unsetCustomStatus} from '@actions/views/custom_status';
 import {setStatus} from '@mm-redux/actions/users';
 import {getTheme} from '@mm-redux/selectors/entities/preferences';
 import {getCurrentUser, getStatusForUserId} from '@mm-redux/selectors/entities/users';
+import {isCustomStatusEnabled, makeGetCustomStatus} from '@selectors/custom_status';
 
 import {logout} from 'app/actions/views/user';
 
 import SettingsSidebar from './settings_sidebar';
 
+const getCustomStatus = makeGetCustomStatus();
+
 function mapStateToProps(state) {
     const currentUser = getCurrentUser(state) || {};
     const status = getStatusForUserId(state, currentUser.id);
 
+    const customStatusEnabled = isCustomStatusEnabled(state);
+    const customStatus = customStatusEnabled ? getCustomStatus(state) : undefined;
```

Comment:
IMO this should be set to `null`, but I do not know exactly how @enahum handles this typically in the mobile repo, so this is (kinda) up to him